morf-core/src/main/java/org/alfasoftware/morf/jdbc/SqlDialect.java

```diff
@@ -745,15 +746,14 @@ protected String schemaNamePrefix(@SuppressWarnings("unused") Table table) {
 

```

Comment:
Was it not possible to also drop all the other schemaNamePrefix() methods?

Reply:
These calls can't be replaced with `tableNameWithSchemaName(TableReference tableRef)`:

```
    statement.append("INDEX ")
      .append(schemaNamePrefix(table))
      .append(index.getName())
```

```
  public Collection<String> renameIndexStatements(Table table, String fromIndexName, String toIndexName) {
    return ImmutableList.of("ALTER INDEX " + schemaNamePrefix(table) + fromIndexName + " RENAME TO " + toIndexName);
  }
```

Given the goal here was not to completely refactor the code I just left that as it is.